src/main/java/duke/tasks/ToDosTask.java

```diff
@@ -0,0 +1,29 @@
+package duke.tasks;
+
+import duke.exceptions.NoSuchCommandException;
+
+public class ToDosTask extends Task {
+    public ToDosTask(String name, boolean done){
+        super(name, done);
+    }
+
+    @Override
+    public String toString() {
+        return String.format("[T][%s] %s", (getDone())? "X" : " ", getName());
+    }
+
+    @Override
+    public String toStringOutput() {
+        return String.format("%s | %s | %s", "T", (getDone())? "X" : "O", getName());
+    }
+
+    public static String getNameInput(String input) {
+        return input.split("todo")[1].strip();
+    }
+
+    public static void isLegitInput(String input) throws NoSuchCommandException {
```

Comment:
Naming it checkInput would have been better, since isLegitInput would be better for a boolean function.